contingency/contingency-api/src/main/java/com/powsybl/contingency/ContingencyElementType.java

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2016, All partners of the iTesla project (http://www.itesla-project.eu/consortium)
+ * Copyright (c) 2018, RTE (http://www.rte-france.com)
```

Comment:
2019 now :)
We should extend, not replace the year: "2016-2019".

Reply:
As the initial copyright is own by iTesla, we should have both iTesla and RTE copyrights

Reply:
done

Reply:
I did it, but I'm not sure that is correct.